pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mango"
+version = "0.0.6"
+authors = [
+    {name = "baobab soluciones", email = "mango@baobabsoluciones.es"}
+]
+description = "Library with a collection of usefull classes and methods to DRY"
+readme = "README.rst"
+urls = {repository = "https://github.com/baobabsoluciones/mango"}
+license = {text = "Apache Software License"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Development Status :: 4 - Beta",
+]
+dependencies = [
```

Comment:
Igual es muy estricto poner el == en las versiones. ¿Quizás podemos ser más laxos?

Reply:
Not a bad idea. I could change all to <= in the other branch after we merge this, ok?

Reply:
Ok